Assets/CohortUnityClient/Scripts/CHSession.cs

```diff
@@ -115,8 +140,181 @@ public class CHSession : MonoBehaviour {
 
     private bool socketConnectionActive = false;
 
+    private string jwtToken = "";
+    private UnityWebRequest cohortUpdateEventRequest;
+
+    private List<CHEpisode> episodesArray;
+
+    private string episodeJson;
+    private Boolean successfulServerRequest;
+
+    [Serializable]
+    public class JwtToken
+    {
+      public string jwt;
+    }
+
+
+    [Serializable]
+    public class CHEpisode
+    {
+      public int episodeNumber { get; set; }
+      public string label { get; set; }
+      public List<CHMessage> cues { get; set; }
+    }
+
+
+
+    string serverToEventUrl(string url) {
```

Comment:
Not sure this name is quite suitable at this point since you call this from the auth function as well, so it's more returning the API URL.

Reply:
serverWithApiUrl ?